docs/running-multiple-instances.md

```diff
@@ -0,0 +1,31 @@
+By default, Watchtower will clean up other instances and won't allow multiple instances monitoring the same containers. It is possible to override this behavior by defining a [scope](https://containrrr.github.io/watchtower/arguments/#filter_by_scope) to each running instance. 
```

Comment:
`won't allow multiple instances monitoring the same containers` -> `won't allow multiple instances running on the same docker host or swarm` maybe?

Reply:
Sounds more correct. Fixed.